### PR TITLE
Personalizes onload chat content

### PR DESCRIPTION
**Pull request by nlumiere**
Sets previously-hard-coded displayed chat information such as chat content and headers to be specific to the logged in user

### CODE REVIEW

**Review by showermat — APPROVED** [review]
Looks good to me!  Since `CHAT_ID` is no longer constant, you may want to lower-case it.

File: proj/src/main/webapp/script.js [hunk 1]
```diff
@@ -23,7 +21,7 @@ if(btnLogin){
         const auth = firebase.auth();
 
         const promise = auth.signInWithEmailAndPassword(emailVal, passVal).then(function(user){
-            window.location.replace("static/chat.html");
+            window.location.replace("chat.html");
```
Comment (leeannachen) [inline proj/src/main/webapp/script.js:24]: how come it doesn't need to go to the static folder to get the chat.html?

Comment (nlumiere) [reply proj/src/main/webapp/script.js:24]: because both welcome.html and signup.html are already in the static folder

Comment (leeannachen) [reply proj/src/main/webapp/script.js:24]: gotcha

**Pull request merged**